tests/test_api_token.py

```diff
@@ -2697,6 +2697,37 @@ def test_42_init_verify_sms_token(self):
 
         delete_policy("verify_toks1")
 
+    def test_43_init_token_with_required_description(self):
+        # set require_description policy with value = 'hotp'
+        set_policy(name="require_description",
+                   scope=SCOPE.ENROLL,
+                   action=["{0!s}=hotp".format(ACTION.REQUIRE_DESCRIPTION)])
+        # try to enroll a hotp token
+        with self.app.test_request_context('/token/init',
+                                           method='POST',
+                                           data={
+                                               "otpkey": self.otpkey,
+                                               "type": "hotp"},
+                                           headers={'Authorization': self.at}):
+            res = self.app.full_dispatch_request()
+            result = res.json.get("result")
+            # a policyerror should be raised because hotp needs a description
+            self.assertEqual(result.get("error").get("message"),
+                             "Description required for hotp token.")
+
+        # try to enroll a totp token
```

Comment:
It would be more useful to test if the hotp-token works when a description is given in the parameters.

Reply:
fixed in 2ca66d18f3cc56625edbacac143bb58cb9a3ee07.